### PR TITLE
feat: enable whitesource scan

### PR DESCRIPTION
**Pull request by MJSiddu**
Ticket:

https://zenhub.ibm.com/workspaces/symposium-workflow-5c49e21b79c5ae72931da0d6/issues/symposium/track-and-plan/25345

### CODE REVIEW

**Review by mxpaspa — APPROVED** [review]

File: .github/workflows/branch_build.yml [hunk 1]
```diff
@@ -55,6 +55,26 @@ jobs:
         echo "/usr/local/kubebuilder/bin" >> $GITHUB_PATH
     - name: Test
       run: make operator/test
+
+    - name: Build
```
Comment (ab-ibm) [inline .github/workflows/branch_build.yml:59]: This part is failing, what is purpose of it, do we really need it?

**Review by ab-ibm — CHANGES_REQUESTED** [review]

File: .github/workflows/branch_build.yml (hunk 1 above)
Comment (MJSiddu) [reply .github/workflows/branch_build.yml:59]: Yes, the first step is building, and then running WS scans. 
Reference: https://github.com/whitesource-ft/ws-examples/tree/main/CI-CD

Comment (ab-ibm) [reply .github/workflows/branch_build.yml]: Looks like it's better now but still token should be updated 
```
Bad org token
```

**Review by ab-ibm — APPROVED** [review]

**Comment by MJSiddu** [comment]
LGTM.

**Review by dacleyra — APPROVED** [review]

**Pull request merged**